### ISSUE TITLE
CHAR() with NEXTVAL() as parameter runs NEXTVAL() twice

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sagarswathi/h2database#414]
```
DROP SEQUENCE IF EXISTS my_seq;
CREATE SEQUENCE my_seq START WITH 64 INCREMENT BY 1;
SELECT CHAR(NEXTVAL('my_seq'));
SELECT CHAR(NEXTVAL('my_seq'));
SELECT CHAR(NEXTVAL('my_seq'));

Output will be:
A
C
E

Output should be:
A
B
C
```

Original issue reported on code.google.com by `DerMais...@gmail.com` on 8 Aug 2012 at 11:52

**Comment by GoogleCodeExporter** [comment]
```
Yes, this is strange. I found the problem and will try to fix it in the next 
release.
```

Original comment by `thomas.t...@gmail.com` on 10 Aug 2012 at 6:52
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 1.3.169
```

Original comment by `thomas.t...@gmail.com` on 9 Sep 2012 at 12:24

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 1.3.169
```

Original comment by `thomas.t...@gmail.com` on 9 Sep 2012 at 12:24

**Comment by GoogleCodeExporter** [comment]
Original comment by `thomas.t...@gmail.com` on 9 Sep 2012 at 12:24
- Changed state: **Fixed**
